From b56d7da9127674e9547b5b5f6070f10c05846b06 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Tue, 1 Jun 2021 15:38:32 +0200
Subject: [PATCH] suricata-bin: add dependency for libcap-ng

In OpenWrt 21.02 and onwards, there is added package libcap-ng and build
system detects and it wants to use it:

Package suricata-bin is missing dependencies for the following libraries:
libcap-ng.so.0

However, there should be way how to disable it, because it can restrict
root permission and libcap-ng uses systemd.
---
 net/suricata-bin/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/net/suricata-bin/Makefile b/net/suricata-bin/Makefile
index e346a13ec..d9ca5baef 100644
--- a/net/suricata-bin/Makefile
+++ b/net/suricata-bin/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=suricata-bin
 PKG_VERSION:=4.0.7
-PKG_RELEASE=4
+PKG_RELEASE=5
 
 PKG_SOURCE:=suricata-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://www.openinfosecfoundation.org/download/
@@ -51,6 +51,7 @@ define Package/suricata-bin
 	+libpthread \
 	+librt \
 	+libpcap \
+	+libcap-ng \
 	$(LUAJIT_DEP) \
 	+libnetfilter-queue \
 	+iptables-mod-nfqueue \
-- 
2.25.1

